dependencies.xml

```diff
@@ -35,7 +35,7 @@ Note all install methods after "main" take
 -->
 <dependencies>
   <main>
-    <h5py>2.10</h5py>
+    <h5py/>
```

Comment:
Not strictly needed by this pull request, but raven is not very dependent on the exact version of h5py.